mathics/builtin/box/graphics3d.py

```diff
@@ -384,14 +375,6 @@ def boxes_to_json(self, leaves=None, **options):
                     "ticks": ticks,
                     "ticks_style": js_ticks_style,
                 },
-                "extent": {
```

Comment:
Can't you just ignore it in the other side?

Reply:
Yes, but why keep it there?

Reply:
Same as above. It will change and should. But not before the Mathics-Django release.

Reply:
Probably a good way to approach this is to start going over the API doc in mathics-threejs-backend review that and then come back here to make it match that specification. 

But all of this is major API upheaval. I'd like to have at least one more stable release before that.